runtimes/8.3/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:22.04
+FROM ubuntu:24.04
```

Comment:
maybe change "python2" to "python3" in this file

maybe replace "jammy" with "noble" in this file

Reply:
maybe change "NODE_VERSION=20" to "NODE_VERSION=22", see https://github.com/nodejs/release#release-schedule

Reply:
Thanks! I have applied your suggestions!